.semaphore/semaphore.yml

```diff
@@ -154,6 +154,10 @@ blocks:
         - name: ruby/rails6-sequel
           commands:
             - "rake app=ruby/rails6-sequel app:test"
+        - name: ruby/rails6-shakapacker
+          commands:
+            - echo "APPSIGNAL_FRONTEND_API_KEY=00000000-0000-0000-0000-000000000000" > ruby/rails6-shakapacker/appsignal_key.env
```

Comment:
To confirm I understood: is this because the key needs to be present, but its value does not matter?

Reply:
For testing it doesn't matter because we're not uploading or sending any errors to the production env.
For manual testing and development you do need to set it to see any data being reported, but the app will fail to boot if the key is missing.